services/wallet/token/token.go

```diff
@@ -705,7 +705,13 @@ func (tm *Manager) GetBalancesByChain(parent context.Context, clients map[uint64
 						return nil
 					}
 
+					if len(res) != len(chunk) {
+						log.Error("can't fetch erc20 token balance", "account", account, "error response not complete")
```

Comment:
Not sure how much info we want to add to the logs, but if this condition is difficult to reproduce it might be helpful to add the expected vs gotten lengths to the error message, perhaps even the contract address